src/components/GoodsList.tsx

```diff
@@ -0,0 +1,19 @@
+export const GoodsList: React.FC<Props> = ({ goods }) => {
+  return (
+    <ul className="goods-list">
+      {goods.map(good => (
+        <li
+          key={good}
+          data-cy="Good"
+          className="goods-list__item"
+        >
+          {good}
+        </li>
+      ))}
+    </ul>
+  );
+};
+
+type Props = {
+  goods: string[]
+};
```

Comment:
Try to keep well-known component structure

```suggestion
type Props = {
  goods: string[]
};

const GoodsList: React.FC<Props> = ({ goods }) => {
  return (
    <ul className="goods-list">
      {goods.map(good => (
        <li
          key={good}
          data-cy="Good"
          className="goods-list__item"
        >
          {good}
        </li>
      ))}
    </ul>
  );
};

export default GoodsList
```